index.html

```diff
@@ -0,0 +1,451 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please consider Implementing the changes requested by the previous reviewer they will enhance your project 🚀